core/src/main/scala/cats/syntax/package.scala

```diff
@@ -45,6 +45,7 @@ package object syntax {
   object show extends ShowSyntax
   object strong extends StrongSyntax
   object traverse extends TraverseSyntax
+  object trySyntax extends TrySyntax
```

Comment:
sorry I missed this last time, there is an `all` object at the top of this list, will you make it inherit `TrySyntax` as well?